### PR TITLE
confirm new commits on target repo

### PR DESCRIPTION
**Pull request by stmh**
This MR implements #124

### CODE REVIEW

File: src/Utilities/Utilities.php [hunk 1]
```diff
@@ -425,4 +426,17 @@ public static function getUserFolder()
...
+     * Check if the force-option is set.
+     *
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     *
+     * @return bool
+     */
+    public static function hasForceOption(InputInterface $input): bool
+    {
+        $option = $input->getOption('force');
+        return is_null($option) || !empty($option);
```
Comment (d34dman) [inline src/Utilities/Utilities.php:440]: Just out of curiosity, doesn't `!empty` check for is_null too? (Is this some kind of performance optimisation)?

Comment (d34dman) [reply src/Utilities/Utilities.php:440]: None the less i would like to keep this, as it is more explicit while reading the code.

**Review by d34dman — APPROVED** [review]
Just minor comment out of curiosity. Code looks good for me.

File: src/Utilities/Utilities.php (hunk 1 above)
Comment (stmh) [reply src/Utilities/Utilities.php:440]: `empty(null)` returns true, yes. Testing for an empty option is a bit tricky with symfony console applications. You need to test for `is_null()` -> no option-value, or a truish option-value with `!empty()`.  That means `--force` has the same meaning as `--force=1`. without the `is_null` part `--force` would be equivalent to `--force=0`

Using true as a default value for the option `force` would make the flag obsolete, as it defaults to true, even if it is not present on the command line.

**Review by d34dman — APPROVED** [review]
